.gitignore

```diff
@@ -3,3 +3,4 @@ _site
 .jekyll-metadata
 /node_modules
 /vendor
+.idea/
```

Comment:
Thanks for your PR! Please consider adding this exception to your workstation's global .gitignore file.

https://www.mashupgarage.com/playbook/git/gitignore.html